tests/testsuite/book_test.rs

```diff
@@ -205,7 +205,8 @@ impl BookTest {
 
     /// Builds the book in the temp directory.
     pub fn build(&mut self) -> &mut Self {
-        let book = self.load_book();
+        let mut book = self.load_book();
+        book.disable_minification();
```

Comment:
Why is this necessary? I'm worried about disabling this in tests, since this isn't testing what is actually generated.

Reply:
Because we do some string search in the tests. So for test code readability, I think it's better to disable it here.

We already check most things are working as expected in GUI tests (both CSS and JS) when minified. So I'm not sure it makes a big difference here.